src/store/couter/CounterState.ts

```diff
@@ -2,21 +2,34 @@
 "use strict";
 import { ReduceState } from "almin-reduce-store";
 import { Counter } from "../../domain/counter/Counter";
+import { Payload } from "almin";
 export interface CounterStateLike {
     count: number;
+    history: Array<number>;
 }
 
 export class CounterState extends ReduceState implements CounterStateLike {
     count: number;
+    history: Array<number>;
+
     constructor(state: CounterStateLike) {
         super();
         this.count = state.count;
+        this.history = state.history;
     }
 
-    update({ counter }: { counter: Counter }): CounterState {
+    update({ counter }: { counter: Counter }) {
         return new CounterState({
             ...this as CounterStateLike,
-            count: counter.count
+            count: counter.count,
+            history: counter.history
         });
     }
+
+    reduce(payload: Payload) {
```

Comment:
ReduceStore should be fixed

<T extends ReduceState>

https://twitter.com/wreulicke/status/849800981927804929